faiss/python/swigfaiss.swig

```diff
@@ -204,23 +204,21 @@ namespace std {
 %include <std_shared_ptr.i>
 
```

Comment:
While we're at it, we might as well ditch the above `std::vector` class declaration, and replace it with `%include <std_vector.i>`.

Reply:
Actually, it seems the SWIG wrappers for `std::vector` do not expose the `data()` method, so we might have to revert.

Reply:
Done